src/App.js

```diff
@@ -3,42 +3,81 @@ import "./App.css";
 var Pokedex = require("pokedex-promise-v2");
 var P = new Pokedex();
 
+// const title = <h1>Pokemon Pokedex Encyclopedia!</h1>;
+// ReactDOM.render(title, document.getElementById('root'));
+
 function App() {
+
+  
   const [pokemon, setPokemon] = useState([]);
   useEffect(() => {
     const getPokemon = async () => {
       var interval = {
         limit: 150,
         offset: 0,
       };
+      
+      
       const pokemonList = await P.getPokemonsList(interval);
+
       if (pokemonList?.results) {
         let ourList = [];
         for (const pokemonItem of pokemonList.results) {
           let pokemon = await P.getPokemonByName(pokemonItem.name);
           if (pokemon) {
             ourList.push(pokemon);
+            
           }
+          
         }
         setPokemon(ourList);
+        
       }
+      
     };
-
+    
     getPokemon();
-  }, []);
+    
+    
+  },
+  []);
+  // const randList = () => Pokedex.sort(() => Math.random() - 0.5);
+  
+
+//   function shuffleArray(array) {
```

Comment:
mate I underSTAAAAAAND what you were doing mate, which actually makes more sense than my way, in that you return random pokemon from our list that we already have rather than make a new call to the api. Actually mate this is better than my way 👍

Reply:
Just need to fix this but we can do that layTAH s omaybe just leave this comment for now, up to you mate